test/graph_rewrites/config_for_grappler_test.cc

```diff
@@ -73,8 +71,8 @@ TEST(GrapplerConfig, RConfig1) {
   ConfigProto config_proto;
   auto backend_name = AttrValue();
   backend_name.set_s("CPU");
-  auto device_config = AttrValue();
-  device_config.set_s("1");
+  auto device_id = AttrValue();
```

Comment:
General: If time permits we can add a test checks for failure if backend or device_id is not specified. The current test check agreement with implementation.

Reply:
Added a test for the failure case